cmd/kubeadm/app/cmd/upgrade/common.go

```diff
@@ -76,7 +72,7 @@ func getK8sVersionFromUserInput(flags *applyPlanFlags, args []string, versionIsM
 
 // enforceRequirements verifies that it's okay to upgrade and then returns the variables needed for the rest of the procedure
 func enforceRequirements(flags *applyPlanFlags, dryRun bool, newK8sVersion string) (clientset.Interface, upgrade.VersionGetter, *kubeadmapi.InitConfiguration, error) {
-	client, err := getClient(flags.kubeConfigPath, dryRun)
+	client, err := upgrade.GetClient(flags.kubeConfigPath, dryRun)
```

Comment:
as `GetClient` was moved to the back end this is the only thing that needed to change.